### PR TITLE
fix(iife/umd): named exports if module wrapped

### PR DESCRIPTION
**Pull request by underfin**

### CODE REVIEW

File: crates/rolldown/tests/rolldown/function/extend/entry-wrapped-cjs-named/artifacts.snap [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+//#region main.js
+var require_main = __commonJS({ "main.js"() {
+	require_main();
+} });
+
+//#endregion
+Object.defineProperty(exports, 'default', {
+  enumerable: true,
+  get: function () {
+    return require_main();
```
Comment (underfin) [inline crates/rolldown/tests/rolldown/function/extend/entry-wrapped-cjs-named/artifacts.snap:63]: Here should be add `default` to `exports`.

File: crates/rolldown_testing_config/src/lib.rs [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
-use rolldown_common::{BundlerOptions, OutputFormat};
+use rolldown_common::{BundlerOptions, OutputExports, OutputFormat};
 use schemars::JsonSchema;
 use serde::Deserialize;
 
 #[derive(Deserialize, JsonSchema, Default)]
 #[serde(rename_all = "camelCase", deny_unknown_fields)]
 pub struct ConfigVariant {
   pub format: Option<OutputFormat>,
+  pub extend: Option<bool>,
```
Comment (hyf0) [inline crates/rolldown_testing_config/src/lib.rs:11]: These new fields should be added in

https://github.com/rolldown/rolldown/blob/d5e30898d9a34413afa524212aee476c1f9e7ca5/crates/rolldown_testing_config/src/lib.rs#L38

, so we could visualize them easily in snapshot.

**Review by hyf0 — APPROVED** [review]

File: crates/rolldown/tests/rolldown/function/extend/entry-wrapped-cjs-named/artifacts.snap [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+Variant: (exports: Named, extend: true, format: Iife, name: "module")
+
+# Assets
+
+## main.js
+
+```js
+(function(exports) {
+
+Object.defineProperty(exports, '__esModule', { value: true });
```
Comment (hyf0) [inline crates/rolldown/tests/rolldown/function/extend/entry-wrapped-cjs-named/artifacts.snap]: Seems wrong. why does not cjs format emit this?

Comment (underfin) [reply crates/rolldown/tests/rolldown/function/extend/entry-wrapped-cjs-named/artifacts.snap]: Fixed it.

**Pull request merged**